django/tasks/register.py

```diff
@@ -0,0 +1,28 @@
+class AlreadyRegistered(Exception):
+    pass
+
+
+class TaskDoesNotExist(Exception):
+    pass
+
+
+class TaskRegistry(object):
+    def __init__(self):
+        self._registry = {}
+
+    def register(self, task):
+        if task.name in self._registry:
```

Comment:
Yes, by default we use fully qualified python path and I'd recommend that, or the `app:task` in docs.